core/domain/takeout_service_test.py

```diff
@@ -1218,6 +1219,26 @@ def test_export_data_for_full_user_when_user_id_is_leaked_fails(
                 log_messages
             )
 
+    def test_export_data_when_user_settings_is_none(self) -> None:
+        user_models.UserSettingsModel(
+            id=self.USER_ID_1,
+            email=self.USER_1_EMAIL,
+            roles=[self.USER_1_ROLE],
+            user_bio='I want to leak uid_abcdefghijabcdefghijabcdefghijab'
+        ).put()
+
+        def _mock_get_user_settings(_: str, strict: bool) -> None: # pylint: disable=unused-argument
+            """Mock get_user_settings function to return None."""
```

Comment:
I don't think this is needed for mock function

Reply:
Updated